lib/analyzer.js

```diff
@@ -96,6 +100,58 @@ var Analyzer = function Analyzer(attachAST,
   this.html = {};
 };
 
+/**
+ * @typedef {Object} LoadOptions
+ * @property {boolean} attachAST Whether underlying AST data should be included.
+ * @property {boolean} noAnnotations Whether `annotate()` should be skipped.
+ * @property {function(string): boolean} filter A predicate function that
+ *     indicates which files should be ignored by the loader. By default all
+ *     files not located under the dirname of `href` will be ignored.
+ */
+
+/**
+ * Shorthand for transitively loading and processing all imports beginning at
+ * `href`.
+ *
+ * In order to properly filter paths, `href` _must_ be an absolute URI.
+ *
+ * @param {string} href The root import to begin loading from.
+ * @param {LoadOptions=} options Any additional options for the load.
+ * @return {Promise<Analyzer>} A promise that will resolve once `href` and its
+ *     dependencies have been loaded and analyzed.
+ */
+Analyzer.load = function load(href, options) {
```

Comment:
Can you rename this? load() on the prototype and load() on the class seems ripe for confusion.

analyze()?